ui/app/AppLayouts/Communities/views/EditAirdropView.qml

```diff
@@ -478,9 +480,10 @@ StatusScrollView {
 
             function openPopup(popup) {
                 popup.parent = addButton
-                popup.x = addButton.width + d.dropdownHorizontalOffset
-                popup.y = 0
-
+                if (!dropdown.bottomSheet) {
```

Comment:
Is this condition needed there (and in other places)? I was testing with and without that condition and there is no difference in placing :thinking:

Reply:
removed!